api/server/controllers/groupController.js

```diff
@@ -20,7 +20,6 @@ exports.create = function(req, res) {
         res.json({
             status: res.statusCode,
             message: "Group created successfully",
-            data: group
```

Comment:
The page re-pulls the groups anyway because it needs to update the whole page when a group is created. Also, even if we change that all we might need this to return would be the generated id. It doesn't make sense to pull data you have just posted; you already have it